tests/platform/test_reboot.py

```diff
@@ -116,4 +183,87 @@ def test_warm_reboot(testbed_devices, conn_graph_facts):
         if "disabled" in issu_capability:
             pytest.skip("ISSU is not supported on this DUT, skip this test case")
 
-    reboot_and_check(localhost, ans_host, conn_graph_facts["device_conn"], reboot_type="warm")
+    reboot_and_check(localhost, ans_host, conn_graph_facts["device_conn"], reboot_type=REBOOT_TYPE_WARMREBOOT)
+
+
+@pytest.fixture(params=[15, 5])
+def power_off_delay(request):
+    """
+    @summary: used to parametrized test cases on power_off_delay
+    @param request: pytest request object
+    @return: power_off_delay
+    """
+    return request.param
+
+
+def _power_off_reboot_helper(args):
```

Comment:
Since this function expects a dictionary argument, it usually a better practice to name the argument like `kwargs`.